src/main/java/com/thebois/listeners/IEventBusSource.java

```diff
@@ -0,0 +1,19 @@
+package com.thebois.listeners;
+
+import java.io.Serializable;
+
+import com.google.common.eventbus.EventBus;
+
+/**
+ * Allows access to an eventbus.
+ */
+public interface IEventBusSource extends Serializable {
```

Comment:
style

I actually didn't know that there was an `@FunctionalInterface`-tag before I saw you use it, maybe it's good practice that we plonk it on our functional interfaces?

Add it back, if you want to. (And probably add it to `IActionSource` and `IMatrixSupplier` as well)

Reply:
Sure, I can do that before merging